src/components/ColorPicker/ColorPreview.tsx

```diff
@@ -15,8 +15,8 @@ export const ColorPreview: FC<ColorPreviewProps> = ({ color }) => {
 
     return (
         <div
-            className="tw-sticky tw-box-content tw-shrink-0 tw-h-2 tw-top-0 tw-bg-white tw-z-20 dark:tw-bg-black-95"
-            style={{ backgroundColor }}
+            className="tw-sticky tw-box-content tw-shrink-0 tw-h-2 tw-top-0 tw-bg-white tw-z-20 dark:tw-bg-black-95 tw-shadow-inner tw-border-l tw-border-t tw-border-r tw-border-line"
+            style={{ backgroundColor, borderTopLeftRadius: '3px', borderTopRightRadius: '3px' }}
```

Comment:
Is there any special reason why we are using 3px here?
The Flyout has a `4px` rounded corners... Maybe we should use 4px here too, shouldn't we?

Reply:
This was added inline because I was having problems with tw-rounded-t borders being more rounded than the outer container's borders, but this seems to have resolved itself with the latest changes, so I've removed the inline CSS.